bson/src/main/org/bson/types/ObjectId.java

```diff
@@ -351,12 +363,29 @@ public String toString() {
         return toHexString();
     }
 
-    // see https://docs.oracle.com/javase/6/docs/platform/serialization/spec/output.html
+    /**
+     * Write the replacement object.
+     *
+     * <p>
+     * See https://docs.oracle.com/javase/6/docs/platform/serialization/spec/output.html
+     * </p>
+     *
+     * @return a proxy for the document
+     */
     private Object writeReplace() {
         return new SerializationProxy(this);
     }
 
-    // see https://docs.oracle.com/javase/6/docs/platform/serialization/spec/input.html
+    /**
+     * Prevent normal serialization.
```

Comment:
We should say **de**serialization, because the `readObject` method does not participate in serialization. This comment is applicable to all other similar changes.

Reply:
Done